setup.py

```diff
@@ -51,18 +51,19 @@ def call_package_config(package,option,optional=False):
     extra_macros.append(('PNG_ENABLED', 1))
 else:
     raise Exception("NO PNG HEADERS FOUND, you need to install libpng-dev")
-    
+
 png_libs = call_package_config("libpng", "--libs", True)
 
 jpg_lib = "jpeg"
```

Comment:
Apologies for messy diff, my editor stripped trailing whitespace, relevant changes here